.github/workflows/crowdin.yml

```diff
@@ -20,22 +20,31 @@ jobs:
     if: github.event_name == 'schedule'
     uses: nuxeo/nuxeo-coldstorage/.github/workflows/crowdin.yml@lts-2023
     secrets: inherit
+    concurrency:
+      group: crowdin
 
   call-workflow-in-lts-2025:
     if: github.event_name == 'schedule'
     uses: nuxeo/nuxeo-coldstorage/.github/workflows/crowdin.yml@lts-2025
     secrets: inherit
+    concurrency:
+      group: crowdin
 
   crowdin:
     runs-on: ubuntu-latest
 
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          ref: lts-2021
```

Comment:
Note here that I prefer to be explicit (and anywhere we reference the branch), rather than parsing `${{ github.ref}}` or using `${{ github.ref_name }}`.